scripts/hid_configurator/modules/dfu.py

```diff
@@ -292,16 +313,22 @@ def __init__(self, dfu_package, dev_fwinfo, dev_board_name, dev_bootloader_varia
                                                                      self.bootloader_api)
 
     @staticmethod
-    def _get_bootloader_api(manifest_str):
+    def _get_bootloader_api(manifest_json):
         bootloader_name = None
-        for b in BOOTLOADER_APIS:
-            if b in manifest_str:
-                if bootloader_name is None:
-                    bootloader_name = b
-                else:
-                    # There can be update only for one bootloader in the package.
-                    assert False
-                    return None
+
+        for f in manifest_json["files"]:
+            VERSION_PREFIX = "version_"
+
+            version_keys = tuple(filter(lambda x: x.startswith(VERSION_PREFIX), f.keys()))
+            if len(version_keys) != 1:
+                return None
+
+            temp_bootloader_name = version_keys[0][len(VERSION_PREFIX):]
+
```

Comment:
consider removing the blank line as you use directly use this variable in the if statement